website/content/docs/networking/service-discovery.mdx

```diff
@@ -0,0 +1,143 @@
+---
+layout: docs
+page_title: Service Discovery
+description: |-
+  Learn about how service discovery works in Nomad to connect workloads.
+---
+
+# Service discovery
+
+Service discovery is the process of retrieving the low-level information
+necessary to access a service, such as its IP and port, via a high-level
+identifier, such as a human-friendly service name.
+
+The service information is stored in a service catalog which provide interfaces
+that can be used to query data. In Nomad there are two service catalogs:
+
+- the **native service discovery** catalog is embedded in Nomad and requires no
```

Comment:
```suggestion
- The **native service discovery** catalog is embedded in Nomad and requires no
```

(also a bunch of other locations where we have bullet points with full sentences)